packages/autotask-client/src/models/autotask.ts

```diff
@@ -33,3 +33,15 @@ export interface Autotask extends Pick<CreateAutotaskRequest, 'name' | 'relayerI
   createdAt?: string;
   codeDigest?: string;
 }
+
+export interface SaveSecretsRequest {
+  deletes: string[];
```

Comment:
should the `delete` be optional?

Reply:
@shahnami note sure if you saw this one as it was after the PR was merged, but it does seem like these should be optional params I think

Reply:
Missed this one, thanks for bringing it to my attention :)

This type was copied from our shared models. Perhaps we _should_ make it optional in Defender too.